core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -853,6 +871,14 @@ class KafkaZkClient private (zooKeeperClient: ZooKeeperClient, isSecure: Boolean
     retryRequestUntilConnected(deleteRequest)
   }
 
+  /**
+   * Checks the PreferredReplicaElection path existence
+   * @return true if PreferredReplicaElection patj exists else false
+   */
+  def preferredReplicaPathExists(): Boolean = {
```

Comment:
Since there are only two uses as far as I can tell, maybe we could remove this and let the two callers use `pathExists` directly?